### PR TITLE
INT-5865 appropriate sonatype product logos

### PR DESCRIPTION
**Pull request by fblancosona**
Include appropriate Sonatype product logos in published helm charts

Jira: https://issues.sonatype.org/browse/INT-5865
jenkins: https://jenkins.ci.sonatype.dev/job/integrations/job/cloud/job/Helm3%20Charts/job/Feature%20Snapshot%20Builds/job/INT-5865-appropriate-sonatype-product-logos/7/

### CODE REVIEW

**Review by jflinchbaugh — APPROVED** [review]
+1 that should work, i'd expect. you have the added bonus (i think) that it might show up as soon as it merges, since you're using the names already expected.  nice.

**Comment by jflinchbaugh** [comment]
i think i see copies of these svg files laying around in the root of the project as well? can those be removed in favor of these you've added in the right location?

File: docs/NexusIQServer_Vertical.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...,0,0,8.44,70.48Z"/><path class="cls-3" d="M15.84,69.72a4.13,4.13,0,0,0,2.85,1.21c1.2,0,1.83-.49,1.83-1.21s-1-1.13-2.06-1.37C17,68,15.33,67.63,15.33,65.8c0-1.36,1.16-2.52,3.24-2.52a4.55,4.55,0,0,1,3.23,1.21l-.7,1.11a3.33,3.33,0,0,0-2.52-1c-1.07,0-1.74.42-1.74,1.1s.92,1,2,1.23c1.48.34,3.24.74,3.24,2.7,0,1.46-1.2,2.6-3.43,2.6a4.86,4.86,0,0,1-3.54-1.32Z"/><path class="cls-3" d="M26.73,63.28c2.55,0,4.14,2,4.14,4.59v.39H24.15A2.79,2.79,0,0,0,27,70.9a3.61,3.61,0,0,0,2.52-1l.74,1a4.78,4.78,0,0,1-3.42,1.27,4.46,4.46,0,0,1-.14-8.92Zm-2.6,3.84h5.19a2.52,2.52,0,0,0-2.59-2.54A2.6,2.6,0,0,0,24.13,67.12Z"/><path class="cls-3" d="M31.8,63.49h1.59v1.3a3.61,3.61,0,0,1,2.78-1.49v1.6a2.29,2.29,0,0,0-.55-.05,3,3,0,0,0-2.23,1.23V72H31.8Z"/><path class="cls-3" d="M36.5,63.49h1.69l2.66,6.67,2.65-6.67h1.71L41.71,72H40Z"/><path class="cls-3" d="M49.17,63.28c2.55,0,4.14,2,4.14,4.59v.39H46.59a2.79,2.79,0,0,0,2.88,2.64,3.63,3.63,0,0,0,2.52-1l.74,1a4.78,4.78,0,0,1-3.42,1.27,4.46,4.46,0,0,1-.14-8.92Zm-2.6,3.84h5.19a2.52,2.52,0,0,0-2.59-2.54A2.6,2.6,0,0,0,46.57,67.12Z"/><path class="cls-3" d="M54.24,63.49h1.59v1.3a3.61,3.61,0,0,1,2.78-1.49v1.6a2.29,2.29,0,0,0-.55-.05,3,3,0,0,0-2.23,1.23V72H54.24Z"/></g></g></svg>
```
Comment (adrianpowell) [inline docs/NexusIQServer_Vertical.svg:1]: do these SVG files need to be referenced anywhere in the project, like in an MD file or will they be referenced later?

**Review by hectorhuol — APPROVED** [review]
+1
The changes look good to me. Just look at the note from @jflinchbaugh to delete the SVG files in the root directory

File: docs/NexusIQServer_Vertical.svg (hunk 1 above)
Comment (fblancosona) [reply docs/NexusIQServer_Vertical.svg:1]: @adrianpowell SVG files are referenced from /charts/nexus-iq/Chart.yaml and /charts/nexus-repository-manager/Chart.yaml

**Review by adrianpowell — APPROVED** [review]

**Pull request merged**